addon/index.js

```diff
@@ -205,6 +205,7 @@ Router.prototype.on = function(name, f) {
 };
 
 Router.prototype.send = function(name, data, addon) {
+  this.mod.port.emit('from-addon-to-web', {type: name, data: data});
```

Comment:
Is there any particular reason this line was moved to the top of the function? If it's order dependent, would be good to insert a comment above this line to that effect.